script/test_queue

```diff
@@ -1,9 +0,0 @@
-#!/usr/bin/env ruby
-$LOAD_PATH << File.expand_path(File.join(File.dirname(__FILE__), "../test"))
-require "test_helper"
-require "test_queue"
```

Comment:
If we delete this file do we still need the `test-queue` gem?

Reply:
Good spot! Have pushed up a commit to remove it 👍